dist/tools/urlcheck/check.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+
+# Copyright 2020 Gunar Schorcht <gunar@schorcht.net>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+if [ -z "$(which wget)" ]; then
+    echo "wget is not installed"
+    exit
+fi
+
+: "${RIOTBASE:=$(cd $(dirname "$0")/../../../; pwd)}"
+cd "$RIOTBASE"
+
+: "${RIOTTOOLS:=${RIOTBASE}/dist/tools}"
+. "${RIOTTOOLS}"/ci/changed_files.sh
```

Comment:
Since the tool is not integrated in the `static-test` target this line is equivalent as testing all files, except the one that are in the EXCLUDE variable. Is it intentional ?